contrib/platform/src/com/sun/jna/platform/win32/Advapi32.java

```diff
@@ -284,6 +301,292 @@ boolean CreateWellKnownSid(int wellKnownSidType, PSID domainSid,
+     *         information, call GetLastError.
+     */
+    boolean GetSecurityDescriptorControl(SECURITY_DESCRIPTOR pSecurityDescriptor, IntByReference pControl, IntByReference lpdwRevision);
+
+    /**
+     * The SetSecurityDescriptorControl function sets the control bits of a security descriptor. The function can set only the control
+     * bits that relate to automatic inheritance of ACEs. To set the other control bits of a security descriptor, use the functions,
+     * such as SetSecurityDescriptorDacl, for modifying the components of a security descriptor.
+     * @param pSecurityDescriptor
+     *              A pointer to a SECURITY_DESCRIPTOR structure whose control and revision information are set.
+     * @param ControlBitsOfInterest
+     *              A SECURITY_DESCRIPTOR_CONTROL mask that indicates the control bits to set.
+     * @param ControlBitsToSet
+     *               SECURITY_DESCRIPTOR_CONTROL mask that indicates the new values for the control bits specified by the ControlBitsOfInterest mask.
+     * @return If the function succeeds, the return value is nonzero. If the
+     *         function fails, the return value is zero. For extended error
+     *         information, call GetLastError.
+     */
+    boolean SetSecurityDescriptorControl(SECURITY_DESCRIPTOR pSecurityDescriptor, int ControlBitsOfInterest, int ControlBitsToSet);
+
+    /**
+     * The GetSecurityDescriptorOwner function retrieves the owner information from a security descriptor.
+     * @param pSecurityDescriptor
+     *          A pointer to a SECURITY_DESCRIPTOR structure whose owner information the function retrieves.
+     * @param pOwner
+     *          A pointer to a pointer to a security identifier (SID) that identifies the owner when the function returns.
+     *          If the security descriptor does not contain an owner, the function sets the pointer pointed to by pOwner
+     *          to NULL and ignores the remaining output parameter, lpbOwnerDefaulted. If the security descriptor contains an owner,
+     *          the function sets the pointer pointed to by pOwner to the address of the security descriptor's owner SID
+     *          and provides a valid value for the variable pointed to by lpbOwnerDefaulted.
+     * @param lpbOwnerDefaulted
+     *          A pointer to a flag that is set to the value of the SE_OWNER_DEFAULTED flag in the SECURITY_DESCRIPTOR_CONTROL
+     *          structure when the function returns. If the value stored in the variable pointed to by the pOwner parameter is
+     *          NULL, no value is set.
+     * @return If the function succeeds, the return value is nonzero. If the
+     *         function fails, the return value is zero. For extended error
+     *         information, call GetLastError.
+     */
+    boolean GetSecurityDescriptorOwner(SECURITY_DESCRIPTOR pSecurityDescriptor, PointerByReference pOwner, BOOLByReference lpbOwnerDefaulted);
```

Comment:
There is a PSIDByReference already in the codebase would that be sensible here for pOwner?